### ISSUE TITLE
Improvement Idea - Tag Roles

### ISSUE THREAD

**Issue by barbafire** [Skater901/wc3connect-notification-bot#1]
Hi Skater. I'm currently using your bot in my server "Outerworld Arena Project", and I've came with an idea:

Insert a command for us, users, that allow to configure roles that the bot will tag when he message about a created game.

Inside some servers, there are roles created for gamemodes, or just a role that allows pinging when new games are hosted. With the implementation of this resouce, we can set which roles the bot will tag when generating his auto-message, and so, notifing players. The tags can be at the end of the generated message.

**Comment by Skater901** [comment]
Nice idea. I'll have a look into it. I assume it's possible, but I don't actually know how to do it.

Also apologies for denying your Discord friend request; I thought you were some random adding me. :P Are you unable to send me a message without being my friend?

**Comment by barbafire** [comment]
Actually I can't send you a message. We are not friends and/or we don't share a server in common and/or your account only accepts messages from known users. I got an error message when I tried.

**Comment by Skater901** [comment]
Yeah looks like I can't set my account to accept messages from anyone. That's annoying. At least I know, now.

I've looked into your request, and it's certainly possible. The only caveat is that to include it as part of the command for setting up the notification (which I think makes the most sense), you can only provide one role. I think that should be fine though, right? I can't see you needing to notify multiple roles about a game, since you can set up one role per game type, and add multiple people to that role.

**Comment by barbafire** [comment]
Just a role is more than enough for most servers. I agree.

**Comment by Skater901** [comment]
I've finally finished this one and deployed it. Let me know if you have any issues with it, and I can re-open this issue if there's bugs. But I think it should all be working, based on my testing.
